RuntimeCrashTests/run-runtime-crash-tests.sh

```diff
@@ -0,0 +1,46 @@
+echo "Running Runtime Crash tests\n"
+
+for folder in $(find . -type d -mindepth 1); do
+    cd "$folder" # navigate to current testing folder
+    
+    echo "Building $folder"
+    ./build.sh || exit 1 # This is not supposed to fail and therefore should fail this test
+    echo "Finished building $folder"
+    
+    echo "Running $folder"
+    source ./run.sh # is expected to set EXPECTED_RETURN_CODE and RETURN_CODE variables
```

Comment:
The variable setting is a little unclean to me (we could just directly check exit codes) but it is very explicit which I like - am OK with this style for now but a future PR might wanna refactor this logic a little